### PR TITLE
Move the command description to a markdown file for man pages

### PR DESCRIPTION
**Pull request by dnephin**
Following up on some concerns from #23825 and #25615.

This changes moves the command "long description" back to a markdown file, so that we can use "`" and so that docs contributors don't have to edit source files.

It still uses the cobra man page generation for the flags and the rest of the structure, but reads the long description from a file.

I've ported over the `volume` commands, which were using the in-lined description, as well as all the `network` commands, and a couple random top-level commands.  If this looks good I can proceed with the rest as part of #26721

### CODE REVIEW

**Comment by dnephin** [comment]
cc @thaJeztah  @vdemeester

File: man/docker-top.1.md [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-    PID      TTY       STAT       TIME         COMMAND
-    16623    ?         Ss         0:00         sleep 99999
-
-
-# HISTORY
-April 2014, Originally compiled by William Henry (whenry at redhat dot com)
-based on docker.com source material and internal work.
-June 2014, updated by Sven Dowideit <SvenDowideit@home.org.au>
-June 2015, updated by Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>
-December 2015, updated by Pavel Pospisil <pospispa@gmail.com>
```
Comment (thaJeztah) [inline man/docker-top.1.md]: Do you know if these "history" lines have any official meaning in man files; iow is it OK to omit them? (just curious)

File: man/src/network/disconnect.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Disconnects a container from a network.
+
+```bash
+  $ docker network disconnect multi-host-network container1
```
Comment (thaJeztah) [inline man/src/network/disconnect.md]: nit while I'm browsing; should be no space before `$`

File: man/docker-top.1.md (hunk 1 above)
Comment (dnephin) [inline man/docker-top.1.md]: I believe this is just a header/section like any other. I took a lot at `man git`, and `man top`, and neither of them have history sections (at least on arch).

**Comment by thaJeztah** [comment]
I really like this! Wondering if we need the nested directories for sub commands (`volumes/ls`), and _possibly_ name it `partials` or so (instead of `src`), but that's just bike shedding, no big issue.

We could even expand on this in future to re-use this for the reference docs

design LGTM, I'm moving this to code review, but /cc @vdemeester as well

**Comment by dnephin** [comment]
> Wondering if we need the nested directories for sub commands

We could use underscore separators instead of directories, but I prefer directories.

> possibly name it partials or so (instead of src)

`partials` to me implies that you could mix them together. It's true that they aren't the complete source. I  like `src`, but if we want to change it, something like `descriptions` might be more appropriate than `partials`.

> We could even expand on this in future to re-use this for the reference docs

Absolutely. I'm going to start setting that up now as part of #26784 I think

**Comment by thaJeztah** [comment]
Thanks! Yeah, the naming is just bike-shedding, and no strong preference on my side, so 👍

**Comment by thaJeztah** [comment]
ping @vdemeester PTAL

**Comment by dnephin** [comment]
ping, rebased again

**Comment by thaJeztah** [comment]
hmf needs another rebase already

ping @vdemeester, ptal !

**Comment by FrenchBen** [comment]
ping @vdemeester for review - @dnephin this needs another rebase it seems :\

**Comment by thaJeztah** [comment]
@dnephin could you rebase?

**Comment by dnephin** [comment]
I've been rebasing this almost every week for a few months. I can do it again, but it seems kind of pointless unless we're ready to merge this sometime soon.

**Comment by dnephin** [comment]
rebased again

**Comment by FrenchBen** [comment]
Can we please get this merged asap?

**Review by vdemeester — APPROVED** [review]
LGTM 🐸
@dnephin could you squash your commit though 👼 (actual commit message are pretty much the same for all commits :sweat:)

**Comment by dnephin** [comment]
Squashed, and build is green

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**